src/site/teasers/event.drupal.liquid

```diff
@@ -1,33 +1,5 @@
-{% comment %}
-Required variable: node - the news story entity
-Optional variable: header - the header level ('h2','h3', etc.) defaults to h4
-Example data:
-     {
-        "title": "Another Test Event",
-        "fieldDate": {
-            "startDate": "2019-03-16 10:00:01 UTC",
-            "value": "2019-03-16T10:00:01",
-            "endDate": "2019-03-16 11:00:00 UTC",
-            "endValue": "2019-03-16T11:00:00"
-        },
-        "fieldDescription": "This gives the user an overview of the event in teaser views",
-        "fieldLocationHumanreadable": "Here",
-        "fieldFacilityLocation": {
-          "entity": {
-            "title": "Westmoreland County VA Clinic",
-            "entityUrl": {
-              "path": "/pittsburgh-health-care/locations/westmoreland-county"
-            }
-          }
-        },
-        "entityUrl": {
-          "path": "/node/98"
-        }
-      }
-{% endcomment %}
-
 {% if node != empty %}
-  {% include "src/site/includes/date.drupal.liquid" with fieldDate = node.fieldDate %}
+  {% include "src/site/includes/date.drupal.liquid" with fieldDate = node.fieldDate fieldDatetimeRangeTimezone = node.fieldDatetimeRangeTimezone %}
```

Comment:
Do you need a comma between variables to declare multiple? I don't think I've ever had to yet so I just don't know lol 🙂

Reply:
Nope. It's the magic of Liquid, man!